doobie/src/test/scala/za/co/absa/db/fadb/doobie/JsonArrayIntegrationTests.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.db.fadb.doobie
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import doobie.implicits.toSqlInterpolator
+import io.circe.Json
+import io.circe.syntax.EncoderOps
+import org.scalatest.funsuite.AnyFunSuite
+import za.co.absa.db.fadb.DBSchema
+import za.co.absa.db.fadb.doobie.DoobieFunction.{DoobieMultipleResultFunction, DoobieSingleResultFunction}
+import za.co.absa.db.fadb.testing.classes.DoobieTest
+
+import za.co.absa.db.fadb.doobie.postgres.circe.implicits.jsonArrayGet
+
+class JsonArrayIntegrationTests extends AnyFunSuite with DoobieTest {
+
+  class InsertActorsJson(implicit schema: DBSchema, dbEngine: DoobieEngine[IO])
+    extends DoobieSingleResultFunction[List[Actor], Unit, IO] (
+      values => {
+        val actorsAsJsonList = values.map(_.asJson)
+        Seq(
+          {
+            import za.co.absa.db.fadb.doobie.postgres.circe.implicits.jsonArrayPut
```

Comment:
Isn't it enough now, to import once and on the top level?

Reply:
It would be enough to import it once on the top of the file alongside other imports had we had only json[] or jsonb[]. In this case though we have both and given the way implicits are resolved by their types the compiler is unable to figure out which one to use. That's why in this particular case the imports have to be scope using the curly braces for individual query fragments.